cirq-core/cirq/ion/ion_device_test.py

```diff
@@ -19,16 +19,20 @@
 import cirq
 import cirq.ion as ci
 import cirq.testing
+import sympy.utilities.matchpy_connector
```

Comment:
Is this needed ?

Reply:
Nope, replaced with `import sympy` instead.